ansible_base/resource_registry/apps.py

```diff
@@ -96,11 +96,9 @@ def proxies_of_model(cls):
 
 
 def _should_reverse_sync():
-    enabled = not getattr(settings, 'DISABLE_RESOURCE_SERVER_SYNC', False)
-    for setting in ('RESOURCE_SERVER', 'RESOURCE_SERVICE_PATH'):
-        if not getattr(settings, setting, False):
-            enabled = False
-            break
+    enabled = getattr(settings, 'RESOURCE_SERVER_SYNC_ENABLED', False)
+    if not getattr(settings, 'RESOURCE_SERVER', False):
+        enabled = False
```

Comment:
I'd argue that if `RESOURCE_SERVER_SYNC_ENABLED` is True and `RESOURCE_SERVER` is not set, this is invalid.

Near-term, the easiest way to properly treat this as invalid would be an error-level log. Likewise for not having the SECRET_KEY defined. I would just add a `logger.error` statement to both of these branches. This could be eliminated if we added a Django check, but that would be over-engineered for the moment.